src/presenter/board-presenter.js

```diff
@@ -20,14 +21,15 @@ export default class BoardPresenter {
   #noPointComponent = null;
 
   #pointPresenter = new Map();
+  #currentSortType = SortType.DEFAULT;
 
   constructor (boardContainer, pointsModel) {
     this.#boardContainer = boardContainer;
     this.#pointsModel = pointsModel;
   }
 
   init = () => {
-    this.#boardPoints = [...this.#pointsModel.points];
+    this.#boardPoints = [...this.#pointsModel.points].sort(sortByDate);
```

Comment:
Не совсем очевидная вещь (см [статью](https://ru.hexlet.io/blog/posts/sovershennyy-kod-zlye-odnostrochniki)), лучше отдельно вызвать сортировку #sortPoints. Второе это то что используешь sortByDate, при смене дефолтной сортировки другой программист может не увидеть что здесь тоже следует заменить 
```suggestion
    this.#boardPoints = [...this.#pointsModel.points];
```